app/src/org/commcare/models/database/DbUtil.java

```diff
@@ -28,6 +28,7 @@
     public final static String orphanFileTableName = "OrphanedFiles";
 
     private static PrototypeFactory factory;
+    private static final String[] packageNames = new String[]{"org.javarosa", "org.commcare", "org.odk.collect"};
```

Comment:
We had to add it back due to not currently having the ability to migrate the keys in a prototype factory. I have an idea of how to do that, and will eventually follow through and remove this.